pages/add/form/creators.js

```diff
@@ -1,21 +1,26 @@
 import React, { Component } from 'react';
 import ReactTags from 'react-tag-autocomplete';
 import Service from '../../../js/service';
+import AutoCompleteInput from './auto-complete-input';
 
-const DELIMITERS = [9,13,188]; // keycodes for tab,enter,comma
-
-export default class Creators extends Component {
-  constructor(props) {
-    super(props);
-    this.state = this.getInitialState();
-  }
-  getInitialState() {
+export default class Creators extends AutoCompleteInput {
+  toSchema(field) {
+    // Model data thusly:
+    //
+    // "related_creators": [{
+    //   "creator_id": number | <id of the creator autocompleted. Will be null if the creator was not autocompleted and was manually typed in>,
+    //   "name": string | <name of the creator typed in, not autocompleted. Will be null if the creator was autocompleted>
+    // }, ...]
```

Comment:
Where does `profile_id` fit in here?

Reply:
I don't know, this refactor was only a refactor - probably good to ask @gideonthomas and @gvn whether that id is still necessary

Reply:
It is, see line 29